apps/docs/package.json

```diff
@@ -57,6 +57,7 @@
     "@supabase/supabase-js": "^2.38.2",
     "common": "*",
     "config": "*",
+    "dat.gui": "^0.7.9",
```

Comment:
ohh i guess this is why package-lock changed

Reply:
@fsansalvadore if this is fine, then good to merge

Reply:
yeah it's the GUI for the sandbox mode

Reply:
nice :D